src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -198,53 +198,67 @@ void CompositionalMultiphaseBase::postInputInitialization()
   FlowSolverBase::postInputInitialization();
 
   GEOS_ERROR_IF_GT_MSG( m_maxCompFracChange, 1.0,
-                        getWrapperDataContext( viewKeyStruct::maxCompFracChangeString() ) <<
-                        ": The maximum absolute change in component fraction in a Newton iteration must be smaller or equal to 1.0" );
+                        viewKeyStruct::maxCompFracChangeString() <<
+                        "The maximum absolute change in component fraction in a Newton iteration must be smaller or equal to 1.0",
```

Comment:
```suggestion
                        "The maximum absolute change in component fraction in a Newton iteration must be smaller or equal to 1.0",
```

and same below and same below and same below and same below and same below and same below and same below and same below and same below